server/graphql/invoice_line/src/mutations/inbound_shipment_line/line/insert.rs

```diff
@@ -118,7 +120,7 @@ impl InsertInput {
             stock_line_id: None,
             barcode: None,
             stock_on_hold: false,
-            campaign_id: None,
+            campaign_id,
```

Comment:
This is getting passed via graphql, but I think need to refactor generate now as campaign_id is getting dropped in service layer, and not actually being inserted when we insert an inbound line.

See behaviour here:


https://github.com/user-attachments/assets/d07ff0fa-9f71-4afc-a8bc-9c794eb16caa